test/conformance/kuttl/generate/validation/clusterpolicy/permissions/same-kind/01-crd.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.4
+    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
I think these are chabged because the controller-gen version used is 0.12.0.

Reply:
Those are not our CRDs, they should not change

Reply:
Done. Thanks for the clarification.